BlueShift-iOS-SDK/InApps/BlueShiftNotificationViewController.m

```diff
@@ -100,21 +100,14 @@ - (UIColor *)colorWithHexString:(NSString *)str {
     return [UIColor colorWithRed:(float)r/255.0f green:(float)g/255.0f blue:(float)b/255.0f alpha:1];
 }
 
-- (void)loadImageFromURL:(UIImageView *)imageView andImageURL:(NSString *)imageURL andWidth:(double)width andHeight:(double)height{
+
+/// Download and load image in imageView
+/// @param imageURL Image url to download the image
+/// @param imageView  assign the downloaded image to imageView
+- (void)loadImageFromURL:(NSString *)imageURL forImageView:(UIImageView *)imageView {
     UIImage *image = [[UIImage alloc] initWithData:[self loadAndCacheImageForURLString:imageURL]];
-    
-    // resize image
-    CGSize newSize = CGSizeMake(imageView.frame.size.width, imageView.frame.size.height);
-    UIGraphicsBeginImageContext(newSize);// a CGSize that has the size you want
-    CGFloat xPosition = (imageView.frame.size.width / 2) - (width / 2);
-    CGFloat yPosition = (imageView.frame.size.height / 2) - (height / 2);
-    [image drawInRect:CGRectMake(xPosition, yPosition, width, height)];
```

Comment:
where are we doing this calculation now?

Reply:
This has been handled at the time of imageView creation.